docs/developer-docs/latest/developer-resources/database-apis-reference/rest/populating-fields.md

```diff
@@ -403,15 +415,18 @@ await request(`/api/articles?${query}`);
 
 #### Deeply populate a dynamic zone with 2 components
 
+Dynamic-zones are highly dynamic content structures by essence.
+
+When populating them you have the choice between applying a unique behavior for all the dynamic zone's components or defining per-component's populate queries using the `on` property.
```

Comment:
```suggestion
When populating dynamic zones, you can choose between:
- a shared population strategy, applying a unique behavior for all the dynamic zone's components 
- or a detailed population strategy, defining per-component populate queries using the `on` property.
```

My only suggestion for this PR (so that we use the same "shared/detailed" wording as in the examples)?

Reply:
Thanks!